src/main/java/io/github/darealturtywurty/turtychemistry/core/integration/ClayAlloyFurnaceRecipeCategory.java

```diff
@@ -0,0 +1,58 @@
+package io.github.darealturtywurty.turtychemistry.core.integration;
+
+import io.github.darealturtywurty.turtychemistry.TurtyChemistry;
+import io.github.darealturtywurty.turtychemistry.common.recipe.ClayAlloyFurnaceRecipe;
+import io.github.darealturtywurty.turtychemistry.core.init.BlockInit;
+import mezz.jei.api.constants.VanillaTypes;
+import mezz.jei.api.gui.builder.IRecipeLayoutBuilder;
+import mezz.jei.api.gui.drawable.IDrawable;
+import mezz.jei.api.helpers.IGuiHelper;
+import mezz.jei.api.recipe.IFocusGroup;
+import mezz.jei.api.recipe.RecipeIngredientRole;
+import mezz.jei.api.recipe.RecipeType;
+import mezz.jei.api.recipe.category.IRecipeCategory;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+
+public class ClayAlloyFurnaceRecipeCategory implements IRecipeCategory<ClayAlloyFurnaceRecipe> {
+    public static final ResourceLocation UID = new ResourceLocation(TurtyChemistry.MODID, ClayAlloyFurnaceRecipe.ID);
+    public static final ResourceLocation TEXTURE_LOCATION =
+            new ResourceLocation(TurtyChemistry.MODID, "textures/gui/clay_alloy_furnace.png"); //?
```

Comment:
Can you instead make the texture in the screen class public and then just reference that here instead.